src/components/LoansByCategory/QuickFilters/QuickFilters.vue

```diff
@@ -114,26 +115,31 @@
 					Reset filters
 				</button>
 			</div>
-			<kv-select
-				id="customizedSortBySelector"
-				:disabled="!filtersLoaded"
-				v-model="sortBy"
-				style="min-width: 180px;"
-				@click.native="trackDropdownClick('sort')"
-			>
-				<option
-					v-for="sortType in filterOptions.sorting"
-					:key="sortType.key"
-					:value="sortType.key"
+			<div id="customizedSortBySelector" class="tw-text-action tw-flex tw-items-center tw-gap-1 tw-text-small">
```

Comment:
Do we want the entire component to have `tw-text-action` or just the sort type name? Currently the "Loan sort" text is green like it can be clicked, but it can't actually be clicked. Or do we want this entire `div` clickable?

Reply:
This is from design; is intended to have the entire component with `tw-text-action` but the behavior of a select element.